ts/fast-sync/types.ts

```diff
@@ -5,11 +5,16 @@ export interface FastSyncSenderChannel {
 }
 export interface FastSyncReceiverChannel {
     streamObjectBatches : () => AsyncIterableIterator<FastSyncBatch>
+    receiveSyncInfo: () => Promise<FastSyncInfo>
 }
 export interface FastSyncInfo {
     objectCount : number
     collectionCount : number
 }
+export interface FastSyncProgress {
+    totalObjectsProcessed: number
+    lastObjectsProcessed: number
```

Comment:
If I understand correctly, this is the amount of items received in the last batch? I don't think we need that for now. And, if we do, I think a more descriptive name would be nice.

Reply:
Correct. Have removed `lastObjectsProcessed` for now if unlikely to need it.